app/views/candidate_interface/application_form/show.html.erb

```diff
@@ -0,0 +1,80 @@
+<% content_for :title, page_title(:application_form) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <%= t('page_titles.application_form') %>
+    </h1>
+
+    <h2 class="govuk-heading-m govuk-!-font-size-27 govuk-!-margin-bottom-2">
+      Courses
+    </h2>
+    <% if @application_form.application_choices.empty? %>
+      <p class="govuk-body"><%= t('application_form.courses') %></p>
+    <% end %>
+    <%= govuk_button_link_to 'TODO: Add course', '#' %>
+
+    <h2 class="govuk-heading-m govuk-!-font-size-27 govuk-!-margin-top-4">
+      About you
+    </h2>
+    <ol class="app-task-list govuk-!-margin-bottom-8">
+      <li class="app-task-list__item">
+        <%= govuk_link_to 'TODO: Personal details', '#', class: 'app-task-list__task-name', 'aria-describedby': 'personal-details' %>
```

Comment:
These links have an `aria-describedby` but there isn't currently any element with the specified `id`, so the attribute isn't doing anything. I'm not sure why they are present in the prototype, as they shouldn't be necessary for the accessibility of this, since it's a simple list of links. I would cautiously remove them for now.

@paulrobertlloyd, do you have context as to why the links have `aria-describedby`  in the prototype?

Reply:
I've stripped `aria-describedby` for now and will merge. If we decide we need this, then we can add it in a later PR.

Reply:
Fine to remove `aria-describedby` for now, but should return once we can mark sections as completed. Here’s the markup for an uncompleted item:

```html
<li class="app-task-list__item">
  <a class="app-task-list__task-name" href="#">Personal details</a>
</li>
```

This will be announced as something like 🗣“Personal details, link”.

Here is the markup once it’s completed:

```html
<li class="app-task-list__item">
  <a class="app-task-list__task-name" href="#" aria-describedby="personal-details-completed">Personal details</a>
  <strong class="govuk-tag app-task-list__task-completed" id="personal-details-completed">Completed</strong>    
</li>
```

This will be announced as something like 🗣“Personal details, completed, visited link”.

Reply:
(I’ll update the prototype to only show `aria-describedby` attribute when an item is marked as being completed).

Reply:
Ah makes sense, thanks Paul! We'll be sure to include this when we start adding in the completed section labels.